shared/common-adapters/plain-input.desktop.js

```diff
@@ -0,0 +1,215 @@
+// @flow
+import * as React from 'react'
+import {getStyle as getTextStyle} from './text.desktop'
+import {collapseStyles, styleSheetCreate} from '../styles'
+
+import type {_StylesDesktop} from '../styles/css'
+import type {InternalProps} from './plain-input'
+
+// A plain text input component. Handles callbacks, text styling, and auto resizing but
+// adds no styling
+class PlainInput extends React.PureComponent<InternalProps> {
+  _input: HTMLTextAreaElement | HTMLInputElement | null
+  _isComposingIME: boolean = false
+
+  static defaultProps = {
+    textType: 'Body',
+  }
+
+  _setInputRef = (ref: HTMLTextAreaElement | HTMLInputElement | null) => {
+    this._input = ref
+  }
+
+  _onChangeTextDone = (text: string) => {
```

Comment:
I think we can collapse these three `_onChangeText*` functions into just `_onChange`

Reply:
haha, good call